src/docs/release-notes/R23/FullNotes.mdx

```diff
@@ -5,12 +5,15 @@
 
 ### Angular
 
--   Updated packages to support Angular Material v13
--   Dropped support for Ionic
+-   Updated packages to support Angular / Angular Material v13.
+-   Dropped support for Ionic.
+-   Added `<blui-toolbar-menu>` component.
```

Comment:
Some potential changes based on the comments above: 

Added `<blui-toolbar-menu>` component to existing component library. 
Deprecated `<blui-dropdown-toolbar>` component.